docs/src/environments/_index.md

```diff
@@ -1,51 +1,86 @@
+You can create environments on demand.
+Each environment is tied to a Git branch.
+With [Bitbucket](../integrations/source/bitbucket.md) and [GitHub](../integrations/source/github.md) integrations,
+you can even get a "development server" automatically for every pull request.
 
-You can have branches that aren't tied to a running instance of your application.
-These are called [inactive environments](../other/glossary.md#inactive-environment).
+You can also have branches that aren't tied to a running instance of your application.
+These are called [inactive environments](#environment-status).
 
 ## Default environment
 
-Your default environment comes from your default branch and is a [Production environment](../administration/users.md#environment-types).
+Your default environment comes from your default branch and is a [production environment](../administration/users.md#environment-types).
+Your project must have a default environment,
+but you can [name it as you want](./default-environment.md).
+
+If you subscribed to a production plan, this environment is your **live site**.
+You can give it a [custom domain name](../domains/steps/_index.md) and a [custom TLS certificate](../domains/steps/tls.md).
+
+## Environment status
+
+Your environments can have one of two statuses:
+
+-   [Active](../other/glossary.md#active-environment):
+    A deployed environment with services and data.
+
+-   [Inactive](../other/glossary.md#inactive-environment):
+    An environment that isn't deployed and has no services or data, only code.
+    For example, when you push a local branch created with Git to your project,
+    it creates an inactive environment.
+
+You can see the status of your environments in the [Console](../administration/web/_index.md) or the [CLI](/administration/cli/_index.md).
+
+{{< codetabs >}}
++++
+title=In the Console
++++
 
-If you subscribed to a production plan, this environment is your **live site**
-and you can give it a [custom domain name](../domains/steps/_index.md) and a custom SSL certificate.
+When you open your project, inactive environments appear lighter in the environment list.
+You also can't select them from the **Environment** dropdown menu.
 
-{{< note >}}
+To check the status of an environment,
+you can also open it and view its information panel.
 
-Your project must have a default environment, but you can give it any name.
-See how to [rename the default environment](./default-environment.md).
+<--->
++++
+title=Using the CLI
++++
 
-{{< /note >}}
+To check the status of all your environments, run the `platform environments` command within your project directory.
 
-## Hierarchy
+{{< /codetabs >}}
+
+Note that you can [change an environment's status](./deactivate-environment.md).
+
+## Organize your environments
+
+### Hierarchy
 
 ![Environment hierarchy](/images/management-console/environments.png "0.5")
```

Comment:
:lady_beetle: This isn't exactly what I get when I have active child environments, but I'm not sure if it's important or a bug or what. Instead of blue branch markers, I see nothing.